react/react1/week2/react1-week2-app/src/App.js

```diff
@@ -0,0 +1,60 @@
+import "./App.css";
+import TodoList from "./components/TodoList.js";
+import Timer from "./components/Timer";
+import { useState } from "react";
+import "./App.css"; // stepping away from inline styling tried in week 1
+
+const todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
+
+const todoExamples = [
+  "Wash Car",
+  "Feed pets",
+  "Pet Pets",
+  "Get some air",
+  "Contemplate on existential matters",
+];
+function App() {
+  const [todoList, setTodoList] = useState(todos);
+
+  function addTodo() {
+    const newTodo = {
+      id: Date.now(), // this way id hopefully will never be the same to other elements
```

Comment:
Indeed, and well done for thinking about this case!
Alternatively, you could get the last todo's id, increment it and use that one